db/seeds.rb

```diff
@@ -5,3 +5,24 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+categories = Category.create([{title: "Apparel"},{title: "Snowboards"}])
+
+apparel = Category.first.items.create([{name: "Hoody",
+                      description: "Keeps you slightly warm",
+                      price: 80},
+
+                     {name: "Sweet Jacket",
+                      description: "Keeps you warm",
+                      price: 240}])
+
+
+snowboards = Category.last.items.create([
+                    {name: "gwar possum",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.<br>Space inside { missing.